test/e2e/providers/azure/azure.go

```diff
@@ -0,0 +1,194 @@
+package azure
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	config "github.com/openshift/api/config/v1"
+	mapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	azureprovider "sigs.k8s.io/cluster-api-provider-azure/pkg/apis/azureprovider/v1beta1"
+
+	"github.com/openshift/windows-machine-config-operator/test/e2e/clusterinfo"
+)
+
+const (
+	defaultCredentialsSecretName = "azure-cloud-credentials"
+	defaultImageOffer            = "WindowsServer"
+	defaultImagePublisher        = "MicrosoftWindowsServer"
+	defaultImageSKU              = "2019-Datacenter-with-Containers"
+	defaultImageVersion          = "latest"
+	defaultOSDiskSizeGB          = 128
+	defaultStorageAccountType    = "Premium_LRS"
```

Comment:
We typically add docs to everything :-) but not sure if this will add any useful info here.

Reply:
These const names are indeed pretty descriptive by themselves. Lmk if you want me to add docs nonetheless.

Reply:
I am fine without comments in this case.